sktime/forecasting/model_selection/_split.py

```diff
@@ -1034,7 +1048,12 @@ class SlidingWindowSplitter(BaseWindowSplitter):
 
     Split time series repeatedly into a fixed-length training and test set.
 
-    For example for `window_length = 5`, `step_length = 1` and `fh = 3`
+    Test window is defined by forecasting horizons
+    relative to the end of the training window.
+    It will contain as many indices
+    as there are forecasting horizons provided to the `fh` argument.
+
+    For example for `window_length = 5`, `step_length = 1` and `fh = np.arange(1, 4)`
```

Comment:
I would replace this by `fh = [1, 2, 3]` to make it clearer (or there may be a low level of confusion about where it starts or ends - 4? 3?)

Reply:
Curious, `fh` is fine with a list of integers, but `cutoffs` is not :(
Changed as requested.

Reply:
that is odd. should there be coercion?

Reply:
https://github.com/alan-turing-institute/sktime/issues/3082